common/persistence/class.SqlPersistence.php

```diff
@@ -14,70 +15,79 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2013 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ * Copyright (c) 2013-2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
  *
+ * @author Lionel Lecaque  <lionel@taotesting.com>
+ * @author Jerome Bogaerts, <jerome@taotesting.com>
  *
  */
 
+use Doctrine\DBAL\Driver\Statement;
+
 
 /**
  * Persistence base on SQL
- *
- * @author Lionel Lecaque  <lionel@taotesting.com>
- * @license GPLv2
- * @package generis
- *
  */
 class common_persistence_SqlPersistence extends common_persistence_Persistence
 {
-
-    /**
-     *
-     * @author "Lionel Lecaque, <lionel@taotesting.com>"
-     * @param string $statement
-     * @param array $params
-     */
-    public function exec($statement, $params = [])
-    {
-        return $this->getDriver()->exec($statement, $params);
-    }
-
-    
     /**
-     * @author "Lionel Lecaque, <lionel@taotesting.com>"
      * @return common_persistence_sql_SchemaManager
      */
     public function getSchemaManager()
     {
         return $this->getDriver()->getSchemaManager();
     }
-    
+
     /**
-     * @author Lionel Lecaque, <lionel@taotesting.com>
      * @return common_persistence_sql_Platform
      */
     public function getPlatForm()
     {
         return $this->getDriver()->getPlatform();
     }
-    
+
     /**
+     * Execute a statement.
+     *
+     * @param string $statement
+     * @param array $params
+     * @param array $types
+     * @return int number of updated rows
+     */
+    public function exec($statement, array $params = [], array $types = [])
```

Comment:
do we need to alter and update any of the tests?

Reply:
after checking nope, that may requires integration test